applications/machine_learning/configuration/thingy53_nrf5340_cpuapp/prj.conf

```diff
@@ -198,19 +168,11 @@ CONFIG_USB_CDC_ACM_LOG_LEVEL_ERR=y
 CONFIG_USB_DEVICE_LOG_LEVEL_ERR=y
 
 ################################################################################
-# Bootloader Configuration
+# Bootloader and MCUmgr (OTA DFU) Configuration
 
-CONFIG_BOOTLOADER_MCUBOOT=y
 CONFIG_MCUBOOT_IMAGE_VERSION="2.0.0+0"
```

Comment:
As I recall correctly, there was an idea to bump this version to 2.3.0, will this be done later?